src/ONE.Models.CSharp/UI/Attribute.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+using ONE.Models.CSharp.Enums;
+
+namespace ONE.Models.CSharp.UI
+{
+    public class Attribute
```

Comment:
Not really clear to me what this class is supposed to represent. It looks like this is supposed to be the UI Definition based on the properties, but with that definition property it isn't clear.

Reply:
Renamed